backend/middlewares/getPermissionsAdapter.js

```diff
@@ -39,9 +39,8 @@
 			accountLevel: function(username) {
 
 				return this.getUser().then(user => {
-
 					if(!user) {
-						return [];
+						throw ResponseCodes.RESOURCE_NOT_FOUND;
```

Comment:
Should the error be thrown by `User.findByUserName` or perhaps `this.getUser` instead?

Having the check here doesn't seem to catch all cases where the teamspace doesn't exist.

E.g. `http://127.0.0.1/api/Bogdan_asdfD/members`